src/proxy.ts

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Sigh... I wish we didn't have to copy and paste this from VS Code and could just leverage their that's already running. But, :+1: